app/screens/Map.js

```diff
@@ -294,19 +291,29 @@ class Map extends Component {
                         title="List"
                         underlayColor='transparent'
                     />
-                    <Icon
-                        reverse
-                        name={Platform.OS === 'ios' ? 'location-arrow' : 'gps-fixed'}
-                        underlayColor='transparent'
-                        type={Platform.OS === 'ios' ? 'font-awesome' : 'material'}
-                        color={theme.neutral}
-                        reverseColor={theme.orange7}
-                        containerStyle={[s.containerStyle,{ position: 'absolute', bottom: 5, right: 5 }]}
-                        size={24}
+                    <Pressable
+                        style={({ pressed }) => [{},s.containerStyle,s.myLocationContainer,pressed ? s.pressed : s.notPressed]}
                         onPress={() => {
                             locationTrackingServicesEnabled ? this.updateCurrentLocation() : this.setState({ showNoLocationTrackingModal: true })
                         }}
-                    />
+                    >
+                        {({ pressed }) => (
```

Comment:
I don't see why we are using a function here.

Reply:
Yeah, it's not necessary. I added it in anticipation of having some `pressed` style on the actual icon. But then I ended up not going that route. I will clean it up.

Reply:
I'm testing and fixing on my end